browser-ext/content/content.js

```
@@ -0,0 +1,419 @@
+            case 'detectForms': {
+                this.detectForms();
+                this.highlightForms();
+                sendResponse({ success: true, forms: this.detectedForms });
+                break;
+            }
+            case 'autoFill':
+                this.autoFillForms(request.cvData);
+                sendResponse({ success: true  });
+                break;
+            case 'performFill':
+            case 'performFill': {
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)